week02/Valid_Parentheses.cpp

```diff
@@ -0,0 +1,29 @@
+#Time Complexity: O(N)
+#Space Complexity:O(1)
+
+
+class Solution {
+public:
+    bool isValid(string s) {
+      stack<char>check;
+        map<char,char>mp;
```

Comment:
```suggestion
        unordered_map<char,char>mp;
```

map is `O(logN)`, unordered_map is `O(1)` in avg.